commands.js

```diff
@@ -876,8 +877,13 @@ var commands = exports.commands = {
 		}
 
 		if (targetUser.confirmed) {
-			var from = targetUser.deconfirm();
-			ResourceMonitor.log("[CrisisMonitor] " + targetUser.name + " was locked by " + user.name + " and demoted from " + from.join(", ") + ".");
+			if (cmd === 'forcelock') {
+				var from = targetUser.deconfirm();
+				ResourceMonitor.log("[CrisisMonitor] " + targetUser.name + " was locked by " + user.name + " and demoted from " + from.join(", ") + ".");
+			} else {
+				this.popupReply("" + targetUser.name + " is a confirmed user. If you are sure you would like to lock them use /forcelock.");
```

Comment:
originally my idea was to have a popup prompt with a button option to lock then which would parse `/forcelock` automatically, but because `popupReply` doesn't support html that didn't work and I guess I just left it as `popupReply`. It's probably less intrusive to make it `sendReply` though, so I'll change it.